06-building-a-program-start-to-finish/setting-up-continuous-integration-(build-server)/Output/README.md

```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: TravisCI
+- **Deliver**: a link to a successful TravisCI build of your `development` branch (it should include test runs)
+- **With**: *independently*
+
+#### Outcome
+- [Travis-Build](https://travis-ci.org/andela-hchukwu/Inverted-Index)
```

Comment:
Great Job!

Please include a verbose reporter so we can see the execution of each test.